### PR TITLE
"About" page refined

### PR DESCRIPTION
**Pull request by Kuzame**
Few updates made for about page, probably final version as is.

### CODE REVIEW

File: my-app/src/About.css [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+  background-color: #fffed1;
+}
+
+#about-jumbotron a {
+  color: #2c7cff;
+}
+
+
+
+/* Is all of this needed after here? I don't see where it is used. */
```
Comment (jonei005) [inline my-app/src/About.css]: Adrian, are the css properties listed after this line all needed? I couldn't really see where they were being used for the most part. If not, might as well delete them to keep it as clean as possible.

Comment (Kuzame) [reply my-app/src/About.css]: They aren't actually, we can delete them if we have time.

**Comment by jonei005** [comment]
Cleaned up the code right now, so someone else please let us know if we should merge yet or if you guys want other changes to be made.

**Comment by lepatrick714** [comment]
I'm okay with it. Just fix the conflicts

**Pull request merged**